### DISCUSSION TITLE
Hausverbrauch aufschlüsseln ...

### DISCUSSION THREAD

**Discussion by schaufma** [solectrus/solectrus#2622] [Ideas]
... mit MQTT Werten und/oder JSON Informationen?

Hallo!
ich habe mehrere Tasmota Steckdosen und auch zwei Heizstäbe. Die Tasmota Teile können eine Verbrauchsmessung (über MQTT) zur Verfügung stellen und die Heizstäbe stellen ein recht ausführliches JSON File zur Verfügung. Die Idee wäre nun, diese Werte in den Bereich "Hausverbrauch" mit aufzunehmen und dort als separate (konfigurierbare) Werte/Balken anzuzeigen.

Also somit den Bereich "Hausverbrauch" etwas aufzuteilen ...  und die Einzelwerte auch sichtbar machen... Alles was nicht ein Einzelwert ist, ist Sonstiger (unbekannter) Verbrauch (der Rest).

Ich hoffe, ich hab das einigermaßen verständlich beschrieben.

Gruß

Markus

**Comment by ledermann** [comment]
Ja, das wird kommen, siehe https://github.com/solectrus/solectrus/issues/2239
